frontend/src/components/Item/MediaStreamSelector.vue

```diff
@@ -151,7 +134,7 @@ if (
 watch(
   trackIndex,
   (newValue) => {
-    emits('input', newValue);
+    emits('input', newValue ?? undefined);
```

Comment:
```suggestion
    emits('input', newValue);
```

Isn't this unnecessary?

Reply:
As newValue can be null and not undefined, and that we need an undefined value in upper layers, yes I need it ^^